docker-cmd.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -eu
+
+export ARCHIVE_DIR=/archive
+export FILES_TO_PUBLISH=/incoming
+
+if [ -z "$CKAN_URL" ]; then
+    exit 1
+fi
+
+if [ -z "$CKAN_API_KEY" ]; then
+    exit 1
+fi
+
+if [ -z "$CKAN_CLIENT_USER_AGENT" ]; then
+    exit 1
+fi
+
+if [ -z "$CKAN_PACKAGE_ID" ]; then
+    exit 1
+fi
+
+if [ -z "$SSH_KEY_FILE" ]; then
+    exit 1
+fi
+
+if [ -z "$SFTP_USER" ]; then
+    exit 1
+fi
+
+if [ -z "$SFTP_HOST" ]; then
+    exit 1
+fi
+
+if [ -z "$SFTP_DIR" ]; then
+    exit 1
+fi
+
+if [ ! -f "$DECRYPTION_KEY" ]; then
```

Comment:
What if we arn't decrypting the data we are picking up?

Reply:
You could use this check to decide if the intention is to run a decrypt process or not.